code/modules/mining/materials.dm

```diff
@@ -205,7 +205,7 @@ var/global/list/initial_materials	//Stores all the matids = 0 in helping New
 /datum/material/plasma
 	name="Plasma"
 	id=MAT_PLASMA
-	value=40
+	value=0.2
```

Comment:
Why is this and diamond's reduction so extreme compared to the other ores.  This is 200 times less valuable compared to iron and sand's time 5 reduction or gold, silver, and uranium's 80 times reduction.

Reply:
Plasma is the most abundant and least useful thing a miner can bring in. While I do see reason in bumping up G/S/U in value, I feel very good where the price of plasma has landed.